docs/docs/twig/ref.md

```diff
@@ -33,3 +35,20 @@ const enhance = withRefs(component => ({
     }
 }));
 ```
+## Populating `refs` of component object
+If you are using `withRefs` and want to access this NodeElement through `component.refs`, you should populate `refs` yourself.
```

Comment:
Can we add a disclaimer that we don't recommend to do this?

Reply:
One problem with storing a ref on a component instance is that it is not accessible in the `lifecycle` higher-order component, as its scope is limited to `el,refs,props,state,dispatch,getState`:
https://github.com/trivago/melody/blob/master/packages/melody-hoc/src/lifecycle.js#L16
What do you recommend?

Reply:
What I've found to work quite well is to dispatch those refs as actions and to catch them with an epic. Avoids pretty much all of the usual issues.